projects/ngx-datatable/src/lib/components/body/ghost-loader/ghost-loader.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: `ghost-loader`,
+  templateUrl: `./ghost-loader.component.html`,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DataTableGhostLoaderComponent {
+
+  @Input() columns;
+  @Input() pageSize;
+
```

Comment:
Minor spacing alignment to keep it in-sync with the rest of the lib (e.g. https://github.com/siemens/ngx-datatable/blob/master/projects/ngx-datatable/src/lib/components/columns/column-cell.directive.ts):

```suggestion
  @Input() columns;
  @Input() pageSize;
```